controller_apache2.yml

```diff
@@ -0,0 +1,49 @@
+- name: Copy config file
+  template: src=files/apache2_ports.conf dest=/etc/apache2/ports.conf
+
+- name: Enable apache modules
+  apache2_module: state=present name="{{ item }}"
+  with_items:
+    - proxy
+    - rewrite
+    - proxy_http
+    - headers
+    - ssl
+
+- name: Install jiocloud ssl certs
```

Comment:
Separate the installation of jiocloud_ssl_certs and all the changes related to it. This will happen in reusability of same.